dogma.go

```diff
@@ -588,3 +634,20 @@ func (c *Client) RepoWatcher(projectName, repoName, pathPattern string) (*Watche
 	rw.start()
 	return rw, nil
 }
+
+// SetMetricCollector sets metric collector for the client.
+// For example with prometheus:
```

Comment:
For example, with Prometheus: (comma and P)

Reply:
🙇 Thank you very much, I will address it asap.